Improve IPIntel reasons

VasilisThePikachu opened the pull request:
Title...

VasilisThePikachu committed:
Improve IPIntel reasons

VasilisThePikachu committed:
Better

PJB3005 commented on Resources/Locale/en-US/connection-messages.ftl:
```diff
@@ -57,8 +57,8 @@ baby-jail-account-reason-overall = Your overall playtime on the server must be y
-ipintel-server-ratelimited = This server uses a security system with external verification, which has reached its maximum verification limit. Please contact the administration team of the server for assistance and try again later.
-ipintel-unknown = This server uses a security system with external verification, but it encountered an error. Please contact the administration team of the server for assistance and try again later.
-ipintel-suspicious = You seem to be connecting through a datacenter or VPN. For administrative reasons we do not allow VPN connections to play. Please contact the administration team of the server for assistance if you believe this is false.
+ipintel-server-ratelimited = This server uses a auditing system with external verification, but the server has reached it's maximum verification limit with the external service. Please contact the administration team of the server to inform them and for further assistance or try again later.
```
```suggestion
ipintel-server-ratelimited = This server uses a auditing system with external verification, but the server has reached its maximum verification limit with the external service. Please contact the administration team of the server to inform them and for further assistance or try again later.
```

PJB3005 commented on Resources/Locale/en-US/connection-messages.ftl:
```diff
@@ -57,8 +57,8 @@ baby-jail-account-reason-overall = Your overall playtime on the server must be y
-ipintel-suspicious = You seem to be connecting through a datacenter or VPN. For administrative reasons we do not allow VPN connections to play. Please contact the administration team of the server for assistance if you believe this is false.
+ipintel-server-ratelimited = This server uses a auditing system with external verification, but the server has reached it's maximum verification limit with the external service. Please contact the administration team of the server to inform them and for further assistance or try again later.
+ipintel-unknown = This server uses a auditing system with external verification, but an error was encountered while verifying your connection. Please contact the administration team of the server to inform them and for further assistance or try again later.
+ipintel-suspicious = You seem to be attempting to connect using a datacenter, proxy, VPN or otherwise suspecious connection. For administrative reasons we do not allow these connections to play. If you have a VPN or similar enabled please turn it off and attempt to reconnect or contact the administration team of the server for assistance on how to continue if you believe this is false or require  ...
```
Needs more commas.

VasilisThePikachu replied:
I hate the English language, where do you like to see more commas?

VasilisThePikachu committed:
Better 2

VasilisThePikachu committed:
Grammarly (aka reviews)

Writing's not that easy, but Grammarly can help. This sentence is grammatically correct, but it's wordy and hard to read. It undermines the writer's message, and the word choice is bland. Grammarly's cutting-edge technology helps you craft compelling, understandable writing that makes an impact on your reader.
